browser/resources/settings/brave_rewards_page/brave_rewards_page.ts

```diff
@@ -131,14 +157,25 @@ class SettingsBraveRewardsPage extends SettingsBraveRewardsPageBase {
     })
   }
 
-  isAutoContributeSupported_() {
-    this.browserProxy_.isAutoContributeSupported().then((supported) => {
-      // Show auto-contribute settings if this profile supports it
-      this.shouldShowAutoContributeSettings_ = supported
-    })
+  private openRewardsPanel_ () {
+    chrome.braveRewards.openRewardsPanel()
+    this.isAutoContributeSupported_()
+  }
+
+  private async isAutoContributeSupported_ () {
```

Comment:
Just by looking at the name, I was expecting this function to return a boolean. Do we want to rename it?

Reply:
Good point (and I meant to do that and then forgot!) Fixed.